src/test/java/com/Maven/java_bootcamp/AppTest.java

```diff
@@ -0,0 +1,23 @@
+package com.Maven.java_bootcamp;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+public class AppTest 
```

Comment:
is missing the last part of the exercise.

> Then skip the unit test by property or by adding the skipping test configuration to your pom.xml file.

Reply:
Skipping unit test configuration was added to pom.xml file.